<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>inserimentoturno_window</class>
 <widget class="QMainWindow" name="inserimentoturno_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>762</width>
    <height>473</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Inserimento Turno</string>
  </property>
  <property name="windowIcon">
   <iconset resource="xz.qrc">
    <normaloff>:/images/images/AnconAmbiente_Logo.png</normaloff>:/images/images/AnconAmbiente_Logo.png</iconset>
  </property>
  <widget class="QWidget" name="inserimentoturno_widget">
   <widget class="QTabWidget" name="inserimentoturno_tabs">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>741</width>
      <height>391</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="dati_tab">
     <attribute name="title">
      <string>Dati</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>240</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Ora Inizio Turno:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>280</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Ora Fine Turno:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Servizio:</string>
      </property>
     </widget>
     <widget class="QTimeEdit" name="time_end_field">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>280</y>
        <width>191</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QTimeEdit" name="time_start_field">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>240</y>
        <width>191</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>205</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Data:</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="date_field">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>200</y>
        <width>191</width>
        <height>28</height>
       </rect>
      </property>
      <property name="tabletTracking">
       <bool>true</bool>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="search_field">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>10</y>
        <width>581</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>10</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/images/images/loupe (1) (1).png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tabella_servizi">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>30</y>
        <width>601</width>
        <height>161</height>
       </rect>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>150</number>
      </attribute>
      <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <row>
       <property name="text">
        <string>DADADAD</string>
       </property>
      </row>
      <column>
       <property name="text">
        <string>ID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Tipo</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Luogo</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Periodicit√†</string>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QWidget" name="mezzi_widget">
     <attribute name="title">
      <string>Mezzi</string>
     </attribute>
     <widget class="QComboBox" name="mezzo1_combo">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>10</y>
        <width>161</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>15</y>
        <width>65</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Mezzo 1:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="aggiungimezzo_button">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>241</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>AGGIUNGI MEZZO</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="operatori_tab">
     <attribute name="title">
      <string>Operatori</string>
     </attribute>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>15</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Operatore 1:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="operatore1_combo">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>10</y>
        <width>161</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="aggiungioperatore_button">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>241</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>AGGIUNGI OPERATORE</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="inserisci_button">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>410</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:rgb(24, 105, 68);
color: white;
border-style: outset;
font:  bold 11px


}

QPushButton:hover{
background-color:rgb(170, 203, 108);
color: white;
border-style: outset;
font:  bold 11px
}</string>
    </property>
    <property name="text">
     <string>INSERISCI</string>
    </property>
   </widget>
   <widget class="QPushButton" name="annulla_button">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>410</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:rgb(24, 105, 68);
color: white;
border-style: outset;
font:  bold 11px


}

QPushButton:hover{
background-color:rgb(170, 203, 108);
color: white;
border-style: outset;
font:  bold 11px
}</string>
    </property>
    <property name="text">
     <string>ANNULLA</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="xz.qrc"/>
 </resources>
 <connections/>
</ui>
